src/consensus/params.h

```diff
@@ -339,6 +382,24 @@ struct Params {
         int startHeight,
         int endHeight);
 
+    std::optional<OnetimeLockboxDisbursement> vOnetimeLockboxDisbursements[MAX_ONETIME_LOCKBOX_DISBURSEMENTS];
+
+    /**
+     * Defines a one-time lockbox disbursement for this network.
+     *
+     * The disbursement amounts are hard-coded, instead of being calculated as the amount
+     * in the lockbox at the upgrade activation. `nChainLockboxValue` tracks the latter,
+     * but we only know it once we've received all of a block's ancestors, which would be
+     * too late in the consensus rules. We need to be able to calculate `lockboxValue` in
+     * `SetChainPoolValues`, at which point we only know the block's height and contents.
+     */
+    void AddZIPXXXLockboxDisbursement(
```

Comment:
```suggestion
    void AddZIP271LockboxDisbursement(
```

I'll fix this in the release PR where we set the testnet params.